engine/access/rpc/backend/backend_test.go

```diff
@@ -860,6 +875,34 @@ func (suite *Suite) TestGetEventsForHeightRange() {
 		suite.Require().Equal(expectedResp, actualResp)
 	})
 
+	// set max height range to 1 and request range of 2
+	suite.Run("invalid request exceeding max height range", func() {
+		headHeight = maxHeight - 1
+		setupHeadHeight(headHeight)
+		blockHeaders = setupStorage(minHeight, headHeight)
+
+		// create handler
+		backend := New(
+			suite.state,
+			suite.execClient,
+			nil, nil,
+			suite.blocks,
+			suite.headers,
+			nil, nil,
+			suite.receipts,
+			suite.chainID,
+			metrics.NewNoopCollector(),
+			nil,
+			false,
+			1, // set maximum range to 1
+			nil,
+			suite.log,
+		)
+
+		_, err := backend.GetEventsForHeightRange(ctx, string(flow.EventAccountCreated), minHeight, minHeight+1)
```

Comment:
Just to double check my understanding, this range is inclusive, so it's both heights minHeight and minHeight + 1 (therefore 2 block IDs)?

Reply:
Exactly https://github.com/onflow/flow-go/blob/512ac6ee47ede2d79d07c89f81755e1b8606873c/engine/access/rpc/backend/backend_events.go#L63